### ISSUE TITLE
Keyboard is in the way in Polls

### ISSUE THREAD

**Issue by moritz-t-w** [nextcloud/android#12164]
### ⚠️ Before posting ⚠️

- [X] This is a **bug**, not a question or an enhancement.
- [X] I've [searched for similar issues](https://github.com/nextcloud/android/issues) and didn't find a duplicate.
- [X] I've written a clear and descriptive title for this issue, not just "Bug" or "Crash".
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Steps to reproduce

1. Create a poll
2. Select a Text field

### Expected behaviour

Either being able to scroll, or the popup shrinking to fit above the keyboard, or the keyboard autohiding.

### Actual behaviour

The keyboard is in the way and I can't reach the button to create the form. The standard keyboard on my phone doesn't have a hide button, so it stays there.

### Android version

n/a

### Device brand and model

n/a

### Stock or custom OS?

Stock

### Nextcloud android app version

3.26.0

### Nextcloud server version

n/a

### Using a reverse proxy?

I don't know

### Android logs

n/a

### Server error logs

```bash
n/a
```

### Additional information

I did not fill out specifics because there's plenty of keyboards that don't have a hide button

**Comment by joshtrichards** [comment]
There isn't any code related to Polls in the Android app. I think this is just a general Polls app mobile web matter. 

Feel free to open a bug in that project's repository:

https://github.com/nextcloud/polls

Though you should be able to close the keyboard via any of the following:

- hitting the back once
- tapping a non-input area

You can also adjust keyboard settings on any phone I've ever encountered to turn the keyboard closer button back on (the setting is in different spots depending on make/model but easy to lookup via your favorite search engine).

**Comment by moritz-t-w** [comment]
yeah I thought hitting back would work too but according to my coworker who had this problem, it did not. Tapping in a non input area didn't work afaik. I will see if they can change the settings. Thanks

**Comment by moritz-t-w** [comment]
@joshtrichards I have made an issue on nextcloud polls but I've just realised that you may have misunderstood my issue. This is not regarding to the polls app but to chat polls, which are completely different according to my other coworker who set it up. We don't even have polls enabled on our server.

**Comment by moritz-t-w** [comment]
![image](https://github.com/nextcloud/android/assets/90388353/573736e7-6ee2-457e-a88f-3da15b715e55)
This is a chat poll (on the desktop app)

**Comment by joshtrichards** [comment]
Oh okay. In that case you're probably encountering this in the Android Talk app. I'll move this issue over there for you:

https://github.com/nextcloud/talk-android/issues?q=is%3Aissue+polls
